test/js/lib/validate-file-content.test.js

```diff
@@ -64,6 +63,13 @@ describe('Client-side file validation', () => {
     expect(summary).toBe('Your file had 1 error')
   })
 
+  it('catches duplicate cell names in cluster file', async () => {
+    mockReadLinesAndType({ fileName: 'cluster_duplicate_cell_names.txt' })
+    const { errors, summary } = await validateFileContent({ name: 'cluster_duplicate_cell_names.txt' }, 'Cluster')
```

Comment:
I'd like to minimize the amount of test data files we have to maintain, as it's tricky to keep track of which files are used where, and makes reviewing test cases difficult.  So I would shorten the file down to 4-5 rows, and then inline the content similar to how we inlined the content in the `catches text file with .gz suffix` 
(And I know this is different than the other cluster files in this test, but I think Eric merged those while I was on vacation :) )